src/sentry/integrations/opsgenie/integration.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+import logging
+from typing import Any, Mapping, Sequence
+
+from django.utils.translation import gettext_lazy as _
+
+from sentry.integrations.base import (
+    FeatureDescription,
+    IntegrationFeatures,
+    IntegrationInstallation,
+    IntegrationMetadata,
+    IntegrationProvider,
+)
+from sentry.pipeline import PipelineView
+
+logger = logging.getLogger("sentry.integrations.opsgenie")
+
+DESCRIPTION = """
+Trigger alerts in Opsgenie from Sentry.
+
+OpsGenie is a cloud-based service for dev & ops teams, providing reliable alerts, on-call schedule management and escalations.
+OpsGenie integrates with monitoring tools & services and ensures that the right people are notified via email, SMS, phone calls,
+and iOS & Android push notifications.
+"""
+
+
+FEATURES = [
+    FeatureDescription(
+        """
+        Manage incidents and outages by sending Sentry notifications to Opsgenie.
+        """,
+        IntegrationFeatures.INCIDENT_MANAGEMENT,
+    ),
+    FeatureDescription(
+        """
+        Configure Sentry rules to trigger notifications based on conditions you set.
```

Comment:
The PD one is 
> Configure rule based PagerDuty alerts to automatically be triggered in a specific service - or in multiple services!

so we can try:

> Configure rule based Opsgenie alerts to automatically be triggered to notify specific teams

Reply:
@leeandher what do you think about "Configure rule based Opsgenie alerts to automatically be triggered for specific teams"? I feel like the version you suggested has too many instances of "to"